tools/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "tools"
+version = "0.1.0"
+edition = "2021"
+
+[features]
+default = ["dpe_profile_p256_sha256"]
+dpe_profile_p256_sha256 = ["dpe/dpe_profile_p256_sha256"]
+dpe_profile_p384_sha384 = ["dpe/dpe_profile_p384_sha384"]
```

Comment:
It doesn't look like we are using any of these features. I think they can  be removed. We will probably want them later when we need to test the other profile though. Do you think its worth removing them?

Reply:
Nevermind, I didn't see that it was passing the profile to the DPE crate.